src/routes/createGame.ts

```diff
@@ -0,0 +1,101 @@
+import { getClient } from '../databaseConnection'
+import { celebrate, Joi, Segments } from 'celebrate'
+import {
+  startingFundsDefault,
+  smallBlindDefault,
+  type newGameInfo,
+} from '../app'
+import { verifyFCMToken } from '../firebase'
+
+import express, { type Router } from 'express'
+const router: Router = express.Router()
+
+router.get(
+  '/createGame',
+  celebrate({
+    [Segments.QUERY]: Joi.object().keys({
+      creatorToken: Joi.string()
+        .required()
+        .min(1)
+        .max(250)
+        .label('creatorToken'),
+      nickname: Joi.string().required().max(20).label('Nickname'),
```

Comment:
I noticed, there is inconsistency in giving labels. In joinGame.ts label is the same as the value name in GET request. I belive that way may be better as it is easier to correct bad request when you know what the GET should look like.